Controllers and services

irminasciezor opened the pull request.

irminasciezor committed:
add models

irminasciezor committed:
add models

irminasciezor committed:
add models

irminasciezor committed:
add repositories and services

irminasciezor committed:
add mappers

irminasciezor committed:
add controllers

projektinzynierskiup commented:
dodaj "/guest" przed endpointami nad controllerami dla sprawniejszego działania security

matizakoo commented:
dodaj "/guest" przed endpointami na controllerach, możesz wsm dodac jakies zmienne statyczne z tymi stringami i tego uzyc

matizakoo committed:
konfiguracja flyway dziala
utworzona zostala implementacja logowania i rejestracji
brakuje walidacji danych na poziomie rejestracji

matizakoo committed:
regex dla rejestracji

matizakoo committed:
walidacja danych podczas logowania, rejestracji, customowe exceptiony

matizakoo committed:
dodanie claimsow do jwt, maly refactor, dodanie metod do userService

matizakoo committed:
usuniecie soutow

matizakoo committed:
edit security class

irminasciezor committed:
migracje, merge, poprawki
